client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -89,6 +90,12 @@ export const getTask = (
 				actionAdvanceToNext: true,
```

Comment:
If you expand the code above, you'll notice a timing for this checklist task. The text for this task is "Your site has been created". This implies that the task is complete, so I don't see why we would ever even show "timing" per the previous or existing logic in `client/my-sites/customer-home/cards/tasks/site-setup-list/current-task-item.jsx`.